test/calculatorTest.js

```diff
@@ -0,0 +1,28 @@
+const chai = require('chai');
+const { assert } = require('chai');
+const calculator = require('../calculator')
+
+
+describe('Test addition', ()=> {
+    it('Should return addition', ()=>{
+        assert.equal(add(2,2), 4)
+    })
+})
+
+describe('Test substraction', ()=> {
+    it('Should return substraction', ()=>{
+        assert.equal(substract(2,2), 0)
+    })
+})
+
+describe('Test Multiplication', ()=> {
```

Comment:
Codacy has a fix for the issue: [Missing space before =>.](https://app.codacy.com/gh/bakersen/code-quality-badges/pullRequest?prid=8108802)
```suggestion
describe('Test Multiplication', () => {
```